packages/core/src/external/carequality/ihe-gateway-v2/saml/security/sign.ts

```diff
@@ -46,12 +44,14 @@ export function signTimestamp({
   xml: string;
   privateKey: crypto.KeyLike;
 }): string {
+  const transforms = ["http://www.w3.org/2001/10/xml-exc-c14n#"];
```

Comment:
updating timestamp for kno2 to only have one transform

Reply:
This will be compliant to other gateways as well? Otherwise, it would be better to have some map or something so we could have specific settings for each GW OID.

Reply:
Yes it works with all other gateways as well. We are at 92% connectivity